LoopUI/ViewModel/CGMStatusHUDViewModel.swift

```diff
@@ -157,9 +168,44 @@ public class CGMStatusHUDViewModel {
         unitsString = unit.localizedShortUnitString
         accessibilityString = accessibilityStrings.joined(separator: ", ")
     }
+
+    func displayStaleGlucoseValue() {
+        self.glucoseValueString = CGMStatusHUDViewModel.staleGlucoseRepresentation
+        self.glucoseValueTintColor = .label
+        self.trend = nil
+        self.glucoseTrendTintColor = .glucoseTintColor
+        self.manualGlucoseTrendIconOverride = nil
+        if let signalLossHighlight = signalLossHighlight {
+            statusHighlight = signalLossHighlight
```

Comment:
[nit] For consistency, consider `self.` prefix here, too.

Reply:
I'll remove them